prometheus/promhttp/http.go

```diff
@@ -37,19 +37,20 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"strconv"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/prometheus/common/expfmt"
-
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/expfmt"
```

Comment:
I guess golint will complain for this (we separate prometheus client - I know, annoying)